projects/vdk-plugins/vdk-oracle/tests/test_plugin.py

```diff
@@ -0,0 +1,190 @@
+# Copyright 2021-2023 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+import os
+from unittest import mock
+
+import pytest
+from click.testing import Result
+from vdk.plugin.oracle import oracle_plugin
+from vdk.plugin.test_utils.util_funcs import cli_assert_equal
+from vdk.plugin.test_utils.util_funcs import CliEntryBasedTestRunner
+from vdk.plugin.test_utils.util_funcs import jobs_path_from_caller_directory
+
+
+# @pytest.fixture(scope="session", autouse=True)
```

Comment:
I suggest split the test into multiple modules to make them more maintainable. E.g 

test_ingest.py test_managed_connection.py

Reply:
There is one connection test and five ingestion tests. If this becomes a problem somehow, we can split them later.